src/interfaces/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bcrs-shared-components/interfaces",
-    "version": "1.0.16",
+    "version": "1.0.17",
```

Comment:
I'm not sure you need to do this one manually. This one should be updated through `Lerna` after a merge. 
If we have made changes it will know that a version bump is required.

Reply:
See the readme for a little bit of context on the process. 
We can go through it together the first time, after a merge.

Reply:
ok, I'll revert all the version numbers 👍

Reply:
reverted all manual package version updates